draft-ietf-teep-architecture.md

```diff
@@ -1347,6 +1347,16 @@ still a threat present in allowing a potentially compromised TAM to install
 any TA it wants; for example, a known-compromised TA or an explicitly
 malicious TA.
 
+There are also threats of hostile or abusive TAMs where a TAM turns to act out of 
+expectation of device administrators, for example, pushing out TAs that contain 
+some data collection or use users' device resources for distributed jobs for a TAM. 
+The mitigation methods for a compromised TAM case above can also apply to these threats.
+
+A Device Administrator may need to be notified when such an incident takes place.
+An Untrusted Application or some software component in the REE may monitor detection
+output from the device's TEEs, and notify the Device Administrator for additional 
+remediations.
```

Comment:
I find this sentence confusing.  What is "detection output"?   And it's not clear to me that there is any such thing in general, so I'd rather omit this sentence and maybe this whole paragraph.

Reply:
This is a possibility similar to the suggested possible mitigation methods 1) - 3) above, which is out-of-scope for this doc to give actual specifications. A notification scheme is out of scope for this doc to notify an administrator. This is purely to mention that such a support should be considered as part of threat mitigation.

Let me remove this for now, and follow up later if anyone raises a request. Thank you for your suggestion here.